actionpack/lib/action_controller/test_case.rb

```diff
@@ -311,6 +313,14 @@ def destroy
       clear
     end
 
+    def fetch(key, default = Unspecified, &block)
```

Comment:
It's been a while since I worked on this, but I don't believe so. If I remember correctly the whole issue I was trying to fix was that the `TestSession#fetch` did not accept as many arguments as the actual session class in rails, so tests could not be run against controllers that used all of the arguments. Can't seem to remember where I got the pattern of specifying `Unspecified`, I think it was either taken from some place else in the Rails code or from the actual `Rack::Session` tests.